packages/contracts/bin/take-dump.ts

```diff
@@ -18,16 +20,28 @@ import { makeL2GenesisFile } from '../src/make-genesis'
   }
 
   const genesis = await makeL2GenesisFile({
```

Comment:
Move defaults to the top of the file as constants w/ descriptions of what they are